Domains/QuantityTakeoffsAspects/QuantityTakeoffsAspects.ecschema.xml

```diff
@@ -120,7 +132,9 @@
     <ECEntityClass typeName="MaterialAspect" description="An aspect that defines the basic properties of a material." >
         <BaseClass>QuantityTakeoffAspect</BaseClass>
         <ECCustomAttributes>
-            <ClassHasHandler xmlns="BisCore.01.00.00"/>
+            <Deprecated xmlns="CoreCustomAttributes.01.00.03">
+                <Description>Use bis:PhysicalMaterial instead.</Description>
+            </Deprecated>
```

Comment:
the PhysicalMaterial does not cover all cases of this QTO - material.weight is not present, are we sure we're comfortable deprecating this class?